Source/Core/DolphinQt/Config/MouseSettingsWidget.cpp

```diff
@@ -0,0 +1,140 @@
+#include "DolphinQt/Config/MouseSettingsWidget.h"
```

Comment:
Missing license header here.

Also please add a newline between the initial include and the ifdef.

Reply:
I'm happy to add the license header, but have y'all considered adding license headers to the linter or something? I honestly thought those had been added automatically as part of the build process.